src/views/admin/NotificationsView.vue

```
@@ -419,92 +971,318 @@ async function loadSlackDestinationStatus() {
-    notificationConfigurations.value ?? []
-  ).map((configuration) => {
-    if (configuration.name !== notificationName) {
-      return configuration;
+    health.value = null;
+    healthError.value = getApiErrorMessage(
+      error,
+      'Notification health is unavailable.',
+    );
+  } finally {
+    if (requestSequence !== healthRequestSequence) {
+      return;
```

Biome (2.4.15)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)